packages/internals/src/get-generators/generatorResolvers/generatorResolvers.ts

```diff
@@ -19,6 +18,5 @@ export type GeneratorResolvers = {
  * as written in the schema, as well as finding their output paths.
  */
 export const generatorResolvers: GeneratorResolvers = {
```

Comment:
I guess we could also remove this map completely now and just use `prismaClientResolver` directly?

Reply:
Yes, good point.